app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,111 @@
+$(document).ready(function() {
+  var flag = true;
+  var flag2 = true;
+  $(".upvote").on("click", function( event ) {
+    var reviewId = event.toElement.className.match(/\d+/)[0];
+    var score = document.getElementById("review-" +
+      reviewId +
+      "-score"
+    ).innerHTML;
+    score = parseInt(score);
+
+    if (flag) {
+      event.preventDefault();
+      event.stopPropagation();
+
+      score++;
+      document.getElementById("review-" + reviewId + "-score").textContent = score;
+
+      // also hide upvote button
+      var post_url = "/reviews/" + reviewId + "/upvotes";
```

Comment:
js syntax standard is without underscores: `postUrl` would probably be best here (unlike Ruby).